### PR TITLE
fix: Fixed CS1591 warnings about missing XML comments

### LINKED ISSUE #66: Warning CS1591 : Missing XML comment for publicly visible type or member 'DirectionEnumExtensions'
my enum is public and I set 
```xml
		<GenerateDocumentationFile>True</GenerateDocumentationFile>
```

then I got this compile warning:
```
0>Direction_EnumGenerator.g.cs(6,25): Warning CS1591 : Missing XML comment for publicly visible type or member 'DirectionEnumExtensions'
0>Direction_EnumGenerator.g.cs(8,95): Warning CS1591 : Missing XML comment for publicly visible type or member 'DirectionEnumExtensions.DisplayNamesDictionary'
0>Direction_EnumGenerator.g.cs(24,95): Warning CS1591 : Missing XML comment for publicly visible type or member 'DirectionEnumExtensions.DisplayDescriptionsDictionary'
0>Direction_EnumGenerator.g.cs(40,30): Warning CS1591 : Missing XML comment for publicly visible type or member 'DirectionEnumExtensions.ToStringFast(Direction, string)'
0>Direction_EnumGenerator.g.cs(58,28): Warning CS1591 : Missing XML comment for publicly visible type or member 'DirectionEnumExtensions.IsDefinedFast(Direction)'
0>Direction_EnumGenerator.g.cs(76,28): Warning CS1591 : Missing XML comment for publicly visible type or member 'DirectionEnumExtensions.IsDefinedFast(string)'
0>Direction_EnumGenerator.g.cs(94,30): Warning CS1591 : Missing XML comment for publicly visible type or member 'DirectionEnumExtensions.ToDisplayFast(Direction, string)'
0>Direction_EnumGenerator.g.cs(112,30): Warning CS1591 : Missing XML comment for publicly visible type or member 'DirectionEnumExtensions.ToDescriptionFast(Direction, string)'
0>Direction_EnumGenerator.g.cs(130,59): Warning CS1591 : Missing XML comment for publicly visible type or member 'DirectionEnumExtensions.GetValuesFast()'
0>Direction_EnumGenerator.g.cs(147,32): Warning CS1591 : Missing XML comment for publicly visible type or member 'DirectionEnumExtensions.GetNamesFast()'
0>Direction_EnumGenerator.g.cs(164,27): Warning CS1591 : Missing XML comment for publicly visible type or member 'DirectionEnumExtensions.GetLengthFast()'
```

how can I solve this?

### PR DESCRIPTION
**Pull request by OhFlowi**
Fixes #66

### CODE REVIEW

**Comment by EngRajabi** [comment]
please fix conflict

File: Supernova.Enum.Generators/EnumGeneratorAttributeHelper.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿namespace Supernova.Enum.Generators;
+
+public static class EnumGeneratorAttributeHelper
```
Comment (EngRajabi) [inline Supernova.Enum.Generators/EnumGeneratorAttributeHelper.cs:3]: Was there a special reason that you separated these features from the SourceGeneratorHelper class?

File: Supernova.Enum.Generators/EnumGeneratorAttributeHelper.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    public const string Content = @"// <auto-generated />
+
+using System;
+using System.CodeDom.Compiler;
+
+namespace " + SourceGeneratorHelper.NameSpace + @"
+{
+    [AttributeUsage(AttributeTargets.Enum)]
+    [GeneratedCodeAttribute(""Supernova.Enum.Generators"", null)]
```
Comment (EngRajabi) [inline Supernova.Enum.Generators/EnumGeneratorAttributeHelper.cs:15]: Does adding Generated Code Attribute make a special change?

Comment (OhFlowi) [reply Supernova.Enum.Generators/EnumGeneratorAttributeHelper.cs:15]: It's a support attribute for tools that use Reflection for analysis. Does the same thing as the auto-generated tag for tools that not uses a Reflection approach.

File: Supernova.Enum.Generators/EnumGeneratorAttributeHelper.cs (hunk 1 above)
Comment (OhFlowi) [reply Supernova.Enum.Generators/EnumGeneratorAttributeHelper.cs:3]: This separation complies with the single responsibility principle and helps to keep the project clear. Based on the file name, it is now clear what this file does without having to open it.

**Comment by OhFlowi** [comment]
Master merged into branch

File: Supernova.Enum.Generators/EnumGeneratorAttributeHelper.cs (hunk 1 above)
Comment (OhFlowi) [reply Supernova.Enum.Generators/EnumGeneratorAttributeHelper.cs:3]: This separation follows the single responsibility principle and helps keep the project organized. Based on the file name, it is now clear what this file does without having to open it.

**Comment by EngRajabi** [comment]
fix with commit
https://github.com/EngRajabi/Enum.Source.Generator/commit/95ab4ef24112b684be60952dd52e727dcd4e4242

**Pull request closed without merging**